src/WebJobs.Script.WebHost/WebScriptHostManager.cs

```diff
@@ -20,6 +20,7 @@
 using Microsoft.Azure.WebJobs.Host.Config;
 using Microsoft.Azure.WebJobs.Host.Loggers;
 using Microsoft.Azure.WebJobs.Host.Timers;
+using Microsoft.Azure.WebJobs.Logging;
```

Comment:
Yeah, the Logger and App Insights insights classes changed with my latest checkin.